Getting_Started.md

```diff
@@ -487,7 +488,74 @@ To enable these workflows, you (or the repository owner) need to configure the f
 2.  **Log in:** Click the "Login with Google" button.
 3.  You should be redirected to the dashboard, where you can start creating agents.
 
----    
+### Creating a Basic Agent
+
+1. **Access Create Agent Page:**
+   - Click "Create New Agent" button on the dashboard, or
+   - Use the "+" button in the navigation bar
+
```

Comment:
Follow on issue: Screen shots. 
OR
Moving this into another doc/wiki page
(That is to say, not in this PR but potential issue/chore to do later.